features/cellular/framework/AT/AT_CellularDevice.h

```diff
@@ -135,6 +135,10 @@ class AT_CellularDevice : public CellularDevice {
 
     virtual CellularContext *get_context_list() const;
 
+    virtual nsapi_error_t set_baud_rate(int baud_rate);
+
+    virtual nsapi_error_t get_baud_rate(int &baud_rate);
```

Comment:
Is this needed, if we can query baudrate then we already know what it is?

Reply:
Right :)